src/core/testers.ts

```diff
@@ -99,6 +99,21 @@ export const formatIs = (expectedFormat: string): Tester => schemaMatches(schema
     && schema.type === 'string'
 );
 
+/**
+ * Only applicable for Controls.
+ *
+ * This function checks whether the given UI schema is of type Control
+ * and if so, resolves the sub-schema referenced by the control and checks
+ * whether the pattern of the sub-schema matches the expected one.
+ *
+ * @param {string} expectedPattern the expected pattern of the resolved sub-schema
+ */
```

Comment:
Also I wouldn't use a pattern in the tester, we can leave this in here as a tester.